src/test/cheffy/steps_test.clj

```diff
@@ -0,0 +1,66 @@
+(ns cheffy.steps-test
+  (:require
+   [cheffy.recipes-test :as recipes-test]
+   [cheffy.util.transit :refer [transit-write]]
+   [clojure.test :refer [deftest is testing]]
+   [test-utils :as tu]))
+
+(def step-id-store (atom nil))
+
+(deftest steps-test
+  #_(testing "list recipes"
+      (testing "with auth -- public and drafts"
+        (let [{:keys [body]} (tu/assert-response-body 200
+                                                      :get "/recipes"
+                                                      :headers {"Authorization" "auth|5fbf7db6271d5e0076903601"})]
+          (is (vector? (get body "public")))
+          (is (vector? (get body "drafts")))))
+      (testing "without auth -- only public "
+        (let [{:keys [body]} (tu/assert-response-body 200 :get "/recipes")]
+          (is (vector? (get body "public")))
+          (is (nil? (get body "drafts"))))))
+  (testing "create step"
+    ;; first create a recipe so we can reference it
+    (let [recipe-id (recipes-test/create-recipe)
+          {:keys [step-id]} (tu/create-entity "/steps"
+                                              {:recipe-id recipe-id
+                                               :description "My first recipe"
+                                               :sort-order 1 ; TODO: not sure what this means
+                                               })]
```

Comment:
notice the usage of `create-entity` instead of repeating myself.
I almost missed this opportunity and noticed it only during self-review on github.